### ISSUE TITLE
Publish a 0.1.0 to crates.io

### ISSUE THREAD

**Issue by mvertescher** [psoc-rs/psoc6-pac#3]
Just a reminder to publish this crate for others! :)

**Comment by jonas-schievink** [comment]
For which PSoC chips does this crate currently work? It only uses the PSoC 63 SVD, does that also work for the 62 and 64? It might be useful to rename the crate to `psoc63-pac` if not.

**Comment by mvertescher** [comment]
Yeah, that's a good point. I'm pretty sure that the 62 and 64 are different so right now this crate only works for the 63. 

I remember from our conversation at RustFest that the other SVD files were missing. I'll take another look and see if I can find them then update the crate accordingly. If not, I'll rename this crate to `psoc63-pac`.

**Comment by jrmoulton** [comment]
Hey @mvertescher I want to generate configs for the psoc4 family but I can't find the svd files. Any idea where those svd files might be?

**Comment by mvertescher** [comment]
Hey, sorry for the late reply. It's been a while, but I _think_ I got them from a Cypress SDK (I forget which one it was though). Anyways, looks like you [found them](https://github.com/jrmoulton/psoc4-pac/tree/main/svd) already 👍

**Comment by jrmoulton** [comment]
Haha yep I was able to find them. Thanks for the reply

**Comment by elamre** [comment]
Still any interest in publishing this on crates.io?
